pyroomacoustics/room.py

```diff
@@ -769,6 +769,7 @@ def __init__(
         humidity=None,
         air_absorption=False,
         ray_tracing=False,
+        use_rand_ism=False,
```

Comment:
Can you please add documentation for the new parameter in the docstring ?

I think it would be great if we can add a section on the method in the documentation at the top of the file. We could even add a figure there.